configuration/pih/scripts/global/mch.js

```diff
@@ -104,3 +104,21 @@ function setUpNextButtonForSections(currentSection) {
     window.htmlForm.submitHtmlForm();
   });
 }
+
+function setupCheckboxRadioGroups() {
+  jq(".checkbox-radio-group").each(function(groupIndex, groupElement) {
+    const options = jq(groupElement).find(".checkbox-radio-option");
+    jq(options).each(function(index1, option1) {
```

Comment:
Any reason to require the `.checkbox-radio-option` classes, and not just apply this to every `input:checkbox` within the `.checkbox-radio-group`?

Reply:
Not really, probably overkill.  It's theoretically possible that a form layout could have other checkboxes nested in here that you wouldn't want affected.